flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/failover/FailoverStrategyLoader.java

```diff
@@ -38,9 +38,12 @@
 	/** Config name for the {@link RestartIndividualStrategy} */
 	public static final String INDIVIDUAL_RESTART_STRATEGY_NAME = "individual";
 
-	/** Config name for the {@link RestartPipelinedRegionStrategy} */
+	/** Config name for the {@link AdaptedRestartPipelinedRegionStrategyNG} */
 	public static final String PIPELINED_REGION_RESTART_STRATEGY_NAME = "region";
 
+	/** Config name for the {@link RestartPipelinedRegionStrategy} */
```

Comment:
A "."  is needed at the end of the statement. 
Also for the other 3 config name comments.

Reply:
fixed